pkg/apiserver/registry/intelligence/throughputanomalydetector/rest.go

```diff
@@ -63,9 +66,50 @@ var queryMap = map[int]string{
 		flowStartSeconds,
 		flowEndSeconds,
 		throughput,
+		aggType,
+		algoType,
 		algoCalc,
 		anomaly
 	FROM tadetector WHERE id = (?);`,
+	aggtadpodQuery: `
+	SELECT
+	    id,
+	    sourcePodNamespace,
+        sourcePodLabels,
+        flowEndSeconds,
+        throughput,
```

Comment:
I think it's the mix of spaces and tabs makes the indentation looks strange here. Could you unify them?